ophyd/utils/epics_pvs.py

```diff
@@ -232,6 +233,34 @@ def _set_and_wait(signal, val, poll_time=0.01, timeout=10, rtol=None,
     TimeoutError if timeout is exceeded
     """
     signal.put(val, **kwargs)
+    wait_for_value(signal, val, poll_time=poll_time, timeout=timeout, rtol=rtol, atol=atol)
+
+
+def wait_for_value(signal, val, poll_time=0.01, timeout=10, rtol=None,
```

Comment:
`set_and_wait` is now internal API. Should this be the same?

Reply:
Need to think on this for a moment. Unlike `set_and_wait`, `wait_for_value` should always have the desired behavior in any context: "wait for the value of this signal to be this value", whereas for `set_and_wait` it had weird interaction specifically with signal `set` methods, particularly when we need to customize `_set_and_wait`. I could see an argument for `wait_for_value` being an overridable method on `Signal` objects, I'm not sure I'm ready to commit to a change like that in this PR? If that's desirable in the future, it could be wise to preemptively internalize this.

Reply:
I don't have a feel for how these might be used, if I'm being honest. The only thing I could say is that it's easier to go from private->public than it is public->private in a library.